docs/references.md

```diff
@@ -1,39 +1,81 @@
 # nfcore/rnafusion: Download references for tools
 
-## 1. Using script
+## 1. Using nextflow helper script
+
+Downloading references manually is a tedious long process. To make the pipeline easier to work with, we provide a script to download all necessary references for fusion detection tools.
+
+> **TL;DR:** Most of the tool references are based on Ensembl database.
+
+```bash
+# This example will not download STAR-Fusion reference and iGenomes
+nextflow run nf-core/rnafusion/download-references.nf --all --outdir <PATH>
+```
+
+For additional optional parameters run:
 
 ```bash
-cd utils && sh download-references.sh <PATH>
+nextflow run nf-core/rnafusion/download-references.nf --help
 ```
 
 ## 2.  Manual download
 
-### STAR-Fusion
+### STAR-Fusion (NCBI)
 
 ```bash
-wget https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz -O GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz
+wget -N -N https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz -O GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz
 tar -xvzf GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz
 ```
 
-> Update the config file to include the directory
+> Update your custom configuration file to include the directory
 
 ```groovy
 params {
   star_fusion_ref = "/path/to/GRCh38_v27_CTAT_lib_Feb092018/ctat_genome_lib_build_dir"
 }
 ```
 
+### STAR-Fusion (Custom Ensembl example)
+
+```bash
+# download all chromosomes from ensembl
+wget ftp://ftp.ensembl.org/pub/release-77/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna.chromosome.{1..22}.fa.gz
+wget ftp://ftp.ensembl.org/pub/release-77/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna.chromosome.{MT,X,Y}.fa.gz
+gunzip -c Homo_sapiens.GRCh38.dna.chromosome.* > Homo_sapiens.GRCh38_r77.all.fa
+
+# download fasta file
+wget ftp://ftp.ensembl.org/pub/release-77/gtf/homo_sapiens/Homo_sapiens.GRCh38.77.chr.gtf.gz
+
+# download
+wget ftp://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.hmm.gz
+gunzip Pfam-A.hmm.gz
+hmmpress Pfam-A.hmm
+
+prep_genome_lib.pl
+  --genome_fa Homo_sapiens.GRCh38_r77.all.fa
+  --gtf Homo_sapiens.GRCh38.77.chr.gtf
+  --pfam_db Pfam-A.hmm
+  --CPU 10
+```
+
+> Update your custom configuration file to include the directory
+
+```groovy
+params {
+  star_fusion_ref = "/path/to/ctat_genome_lib_build_dir"
+}
+```
+
 ## Fusioncatcher
 
 ```bash
-wget http://sourceforge.net/projects/fusioncatcher/files/data/human_v90.tar.gz.aa
-wget http://sourceforge.net/projects/fusioncatcher/files/data/human_v90.tar.gz.ab
-wget http://sourceforge.net/projects/fusioncatcher/files/data/human_v90.tar.gz.ac
-wget http://sourceforge.net/projects/fusioncatcher/files/data/human_v90.tar.gz.ad
+wget -N -N http://sourceforge.net/projects/fusioncatcher/files/data/human_v90.tar.gz.aa
```

Comment:
Only one -N should be enough

Reply:
Solved https://github.com/nf-core/rnafusion/commit/90dee5eabaa9b095edbf45196ccc74f4a917077c